css/style.css

```diff
@@ -266,11 +266,6 @@ a:hover {
     background-color: #e0f2f7;
 }
 
-.timeslot.active p:first-child {
```

Comment:
This would now be `#timeslot_active h3`, seems it slipped through when we changed to the ids. The idea was to also color the headline of the active timeslot (see picture).

<img width="650" alt="screen shot 2018-03-03 at 18 39 08" src="https://user-images.githubusercontent.com/1836815/36937358-33f76acc-1f12-11e8-8a58-272b070a0e80.png">

I am fine with removing it, but haven't cherry-picked it yet.